cli/.js/src/main/scala/aqua/run/RunConfig.scala

```diff
@@ -8,6 +9,7 @@ case class RunConfig(
   logLevel: Level,
   printAir: Boolean,
   secretKey: Option[Array[Byte]],
+  services: Map[String, ArgGetterService],
```

Comment:
Shouldn't it be called `arguments`? `RunConfig.services` sounds like "list of configured services"